Directory.Build.targets

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
+  <Import Project="$(RepositoryEngineeringDir)Packages.props" />
```

Comment:
A props usually is usually imported by another props file, in this case `Directory.Build.props`. It's quite uncommon to import a props file in a targets file.

Reply:
I ever imported into `Directory.Build.props`, but it failed to get the specified package version using the following item in sub .props file.
```
<PackageReference Update="Verify.Xunit" Version="19.6.0" />
```
Importing into `Directory.Build.targets` works well. I don't know why it doesn't work in `Directory.Build.props` ...

Reply:
OK, msbuild does it that way as well: https://github.com/dotnet/msbuild/blob/7711999070e51442c3e527f576159828f01cb3bf/Directory.Build.targets#L28

Just ignore my comment, I think it's fine. The reason why this doesn't work in a Directory.Build.props file is because at that time, those PackageReference items aren't yet created and hence can't be updated.